src/tree/tree.py

```diff
@@ -0,0 +1,21 @@
+import argparse , sys , os
+
+def visit(path, depth, max_depth):
+    files = os.listdir(path)
+    for file in files:
+        printLength(depth +1)
+        print(file)
+        if os.path.isdir(path+file) and depth < max_depth:
```

Comment:
use os.path.join(path, file) to concatenate directory paths, ensuring compatibility across different platforms.
```python
full_path = os.path.join(path, file)
if os.path.isdir(full_path) and depth < max_depth:
```